### ISSUE TITLE
o → f?

### ISSUE THREAD

**Issue by dy** [spectjs/spect#181]
Object store creates interference with `v({ a: x, b: y})`, which can be easily observed via `f`.
`o` is not much useful as setter, neither as subscribable - it acts better as props/attrs observer for existing objects, instead of creating new stuff.

For generic subscribing to source there is `f`. `o` could be transformed therefore as
```js
f(el, ({x, y, z}) => {
  // x, y and z are observed props
})
```
(which is curiously very similar to `$(el, el => {})`.)
That whole props observables is whole conceptual layer, not 100% fully required. There is a way to observe object with props which are observables. Lists - we can manage to observe list length as deps too. We can even turn input objects with props into observable-props, same as `o` does.
So basically, make `f([]) === o([])` etc, reduce that difference.
Then object setter will be defined as
```js
let xs = f({}, o => o, function (newValues) { return Object.assign(this, newValues)})
xs({x: 1, y: 2})
```
How to observe new props?
We can limit observed props to only the ones defined in init `let o = v({x: null, y: null}); o.x = 1`.
Basically, `f` can embrace `o` and return observable instead of proxy (the use of proxy is not 100% proven).

**Comment by dy** [comment]
Overridden with #182
